maintainers/maintainer-list.nix

```diff
@@ -3101,6 +3145,7 @@
   };
   endocrimes = {
     email = "dani@builds.terrible.systems";
+    matrix = "@danielle:fairydust.space";
```

Comment:
Please remove this - I'm not an active matrix user outside of more social nix conversations right now, which fluctuates with how busy I am with other things.

It's kinda rude to advertise a way of reaching someone that they didn't volunteer themselves (regardless of whether it could be found in another way).

Reply:
Done.